packages/admin/cms-admin/src/index.ts

```diff
@@ -76,6 +77,7 @@ export type { SiteConfig } from "./sitesConfig/SitesConfigContext";
 export { SitesConfigProvider } from "./sitesConfig/SitesConfigProvider";
 export { useSiteConfig } from "./sitesConfig/useSiteConfig";
 export { useSitesConfig } from "./sitesConfig/useSitesConfig";
+export { useUser } from "./user/useUser";
```

Comment:
I wouldn't export this as public api as it only loads name+role and can't be extended in any way. I would even say there should not be a useUser as private api in cms-admin - couldn't you make a standard gql request just in the component where you need the user?